lib/livebook/runtime/evaluator/doctests.ex

```diff
@@ -161,9 +161,12 @@ defmodule Livebook.Runtime.Evaluator.Doctests do
         &diff_formatter/2
       )
 
-    expected = diff[:right]
-    got = diff[:left]
-    source = String.trim(reason.doctest)
+    {expected, got, source} =
+      if reason.doctest == ExUnit.AssertionError.no_value() do
+        {nil, nil, nil}
```

Comment:
We can still compute left and right, no?

Reply:
It is best if we can show as much information about the failure as possible :)

Reply:
yes, i think so!

Reply:
Sounds good, done!